### DISCUSSION TITLE
Another Typo Issue I think probably maybe hopefully

### DISCUSSION THREAD

**Discussion by Takodachi6969** [Part-II-Computational-Physics/.github#28] [General]
I hope that i am correct in saying there is another typo in the equation
The next step is to split this Hamiltonian into a sum of local terms, centred on the bonds between spins:
$$ {\cal H}_{\mathrm{Ising}} = \sum_{i=1}^N {\cal H}_{i, i+1}^{\mathrm{loc}} $$
with
$$ {\cal H}_{i,i+1}^{\mathrm{loc}} = - \sigma_i^z \sigma_{i+1}^z + \frac{1}{2}h(\sigma_i^x + \sigma_{i+1}^x)$$.

Where 
$$ {\cal H}_{i,i+1}^{\mathrm{loc}} = - \sigma_i^z \sigma_{i+1}^z - \frac{1}{2}h(\sigma_i^x + \sigma_{i+1}^x)$$. 
should have been a minus which should solve a ton of problems maybe, I hope

**Comment by JNHallen** [comment]
This was indeed another type. I am very sorry about that! Thank you for pointing it out. The template has been updated now.

**Reply by benpymer** [reply]
Hi! I successfully cloned the repo this morning after you updated the template, but the typos were still in my file. Is it possible that you can still get the old template by accident when you clone the repo? Or has the template not properly updated? (Sorry don't know much about GitHub)

**Reply by JNHallen** [reply]
Hi! I am not sure why that would happen to be honest. When I clone the template it gives me the latest version. Perhaps try again?  Luckily the typos have not been too major so far (fingers crossed there are no bigger issues), so perhaps you can also work from the earlier version you got?

**Reply by benpymer** [reply]
I can definitely work from the version I have because I think I've fixed all typos now, and if I haven't I'm sure someone will bring them up on here, so if you get the right template when you clone it then I'm sure it's completely fine! Thanks very much
